helix-core/src/syntax/span.rs

```diff
@@ -67,6 +71,79 @@ pub fn span_iter(spans: Vec<Span>) -> impl Iterator<Item = HighlightEvent> {
     }
 }
 
+impl SpanIter {
+    fn start_span(&mut self, span: Span) {
+        use HighlightEvent::*;
```

Comment:
We can just import the enum variants for the whole crate I think, since a span is just a convenience wrapper around them

Reply:
fixed in 20302313b5d274d231c8c2e505a5306601c0173d